Document typo: "cmake -G ninja..." should be "cmake -G Ninja"

There is a typo in the **cmake build command** this [wiki](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md).

```
cmake -G ninja -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
```

and the next one

```
cmake -G ninja -DCMAKE_BUILD_TYPE=Release -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
```

where `cmake -G ninja` should be `cmake -G Ninja` (`N` capitalized)

Otherwise, you will see this error.

```
CMake Error: Could not create named generator ninja
```

--- a/docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md
+++ b/docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md
@@ -81,7 +81,7 @@ To build debug enclaves:
 cd openenclave
 mkdir build/x64-Debug
 cd build/x64-Debug
-cmake -G ninja -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
+cmake -G Ninja -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
 ninja
 ```
 
@@ -97,7 +97,7 @@ Similarly, to build release enclaves, specify the flag
 cd C:/openenclave
 mkdir build/x64-Release
 cd build/x64-Release
-cmake -G ninja -DCMAKE_BUILD_TYPE=Release -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
 ninja
 ```
 

